authorizer.go

```diff
@@ -142,6 +143,29 @@ func (a *Authorizer) GinTokenAndBackAuthMiddleware() gin.HandlerFunc {
 	}
 }
 
+func (a *Authorizer) GinTokenAndBackAuthMiddlewareV2() gin.HandlerFunc {
```

Comment:
Let's create a generic http version for the chi services!
also, maybe TokenAndApiKey?

Reply:
ok, done